### PR TITLE
🌐 [i18n-KO] Translated `timesformer.md` to Korean

### PR DESCRIPTION
**Pull request by mreraser**
# What does this PR do?

Translated the `timesformer.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

May you please review this PR?
@yijun-lee, @mreraser, @jungnerd, @cjfghk5697

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

Hello @stevhliu !  May you please review this PR? Thank you! 
Have a nice day :)

### CODE REVIEW

**Comment by mreraser** [comment]
Hello! @yijun-lee, @jungnerd, @cjfghk5697
May I kindly ask for your review? Thank you for your consistent reviews 🤗

**Comment by cjfghk5697** [comment]
LGTM!

**Comment by Rocketknight1** [comment]
cc @stehvliu!

**Review by stevhliu — APPROVED** [review]
Nice, thank you!

File: docs/source/ko/_toctree.yml [hunk 1]
```diff
@@ -655,6 +655,11 @@
       - local: in_translation
         title: (번역중) XLSR-Wav2Vec2
       title: (번역중) 오디오 모델
+      - isExpanded: false
+      sections:
+      - local: model_doc/timesformer.md
+        title: TimeSFormer
```
Comment (stevhliu) [inline docs/source/ko/_toctree.yml:661]: I think your alignment may be off here. Should look like:

```yml
- isExpanded: false
  sections:
  - local: model_doc/timesformer
    title: TimeSFormer
```

Comment (mreraser) [reply docs/source/ko/_toctree.yml]: Thank you! I might have missed that. 😅

Comment (stevhliu) [reply docs/source/ko/_toctree.yml]: Oops looks like the toctree still isn't properly aligned! The line [here](https://github.com/mreraser/transformers/blob/9ad85130c6e2c3636642418fca7c80d9aa339035/docs/source/ko/_toctree.yml#L658) should be aligned with the one [here](https://github.com/mreraser/transformers/blob/9ad85130c6e2c3636642418fca7c80d9aa339035/docs/source/ko/_toctree.yml#L663)

Comment (mreraser) [reply docs/source/ko/_toctree.yml]: @stevhliu I have completed modifying the toctree, thank you!

**Pull request merged**